lib/facility.rb

```diff
@@ -1,14 +1,142 @@
+# custom errors site link = https://www.honeybadger.io/blog/ruby-custom-exceptions/
+
+class WrittenTestAlreadyTakenError < StandardError
+
+  attr_reader :message
+  def initialize(msg="User has already taken the written test")
+    @message = msg
+    super(@message)
+  end
+end
+
+class RoadTestAlreadyTakenError < StandardError
+  attr_reader :message
+  def initialize(msg="User has already taken the road test")
+    @message = msg
+    super(@message)
+  end
+end
+
+class VehicleAlreadyRegisted < StandardError
+  attr_reader :message
+  def initialize(msg="This vehicle is already registered")
+    @message = msg
+    super(@message)
+  end
+end
```

Comment:
I like this, but I think it could help be more reusable if we had one CustomError class and then it can take in an argument of the message since the structure of these classes is the same. Regardless, great job playing with this!!!!